common/v2/features/SendAssets/components/SendAssetsForm.tsx

```diff
@@ -239,7 +235,7 @@ export default function SendAssetsForm({
                 <ENSStatus
                   ensAddress={values.receiverAddress}
                   isLoading={isResolvingENSName}
-                  rawAddress={recipientResolvedENSAddress ? recipientResolvedENSAddress! : ''}
+                  rawAddress={values.resolvedENSAddress ? values.resolvedENSAddress : ''}
```

Comment:
I have a hard time understanding this.
Why do we have 2 components to display the address, and why would we give the `receiverAddress` value to the `ensAddress` prop, then set the `rawAddress` prop to the `ensAddress` value?
Can we jump on a call to help me understand?
```
     ensAddress={values.receiverAddress}
     isLoading={isResolvingENSName}
     rawAddress={values.resolvedENSAddress ? values.resolvedENSAddress : ''}
```